src/TouristInfoCards.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function TouristInfoCards(props) {
+  return (
+    <div className="col-md-4">
+      <div className="card mb-4">
```

Comment:
As I mentioned in one of my previous comments, line 5 to 15 could be a `Card` component that is rendered here using map